docs/Buttons.md

```diff
@@ -298,7 +298,20 @@ const ListActions = () => (
 
 ## Record Buttons
 
-### `<DeleteButton>`
```

Comment:
please don't delete this, it's a reminder ;)

Reply:
I put it back